src/en/howto-windows.md

```diff
@@ -0,0 +1,85 @@
+## This guide is currently experimental and under dev
+
+Juju is capable of orchestrating Windows workloads thanks to the enablement
+efforts of [CloudBase Solutions](http://cloudbaseit.com).
+
+> Getting started with orchestrating windows workloads today is still a work in
+progress - as it's only available on 2 substrates. MAAS 1.8 experimental, and
+OpenStack with a custom image loaded in Cinder.
```

Comment:
Thats great - but i cannot today juju deploy windows2012r2/myservice on amazon. its not in streams, therefore its not supported afaict :(